dev/com.ibm.ws.security.spnego/src/com/ibm/ws/security/spnego/internal/SpnegoConfigImpl.java

```diff
@@ -180,6 +192,20 @@ protected String processKrb5Keytab(Map<String, Object> props) {
      */
     protected String processKrb5Config(Map<String, Object> props) {
         String krbCf = (String) props.get(KEY_KRB5_CONFIG);
+        Path kerbConfigFile = kerbSvc.getConfigFile(); // from the <kerberos> element
```

Comment:
We have variable name prefix with krb and kerb ; Can we use one?. I am ok with either one but like to see one prefix for Kerberos.